pkgs/by-name/re/resnap/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+  writeShellApplication,
+}:
+writeShellApplication {
```

Since we're using `writeShellApplication`, we don't have a `version` or `src` attributes, but they're pretty useful to have, so we can do something like this:

```suggestion
let
  name = "reSnap";
  version = "2.5.2";

  src = fetchFromGitHub {
    owner = "cloudsftp";
    repo = "reSnap";
    rev = "v${version}";
    hash = "sha256-P+q5NMGtXE2XCcGGHZJGw8RoOqJSHSv7oyMLkn/ltfY=";
  };
in
writeShellApplication {
  inherit name;

  runtimeInputs = [
    feh
    ffmpeg
    imagemagick_light
    lz4
  ];

  text = "${src}/reSnap.sh";

  meta = {
    description = " Take screnshots of your reMarkable tablet over SSH";
    homepage = "https://github.com/cloudsftp/reSnap";
    license = with lib.licenses; [ mit ];
    maintainers = with lib.maintainers; [ _404wolf ];
    mainProgram = "reSnap";
  };
}
```

Also, the latest git revision seems to be the same as the latest tagged release (2.5.2), so we should use that, instead. 

Can you update the commit message and the PR title to reflect this?